eng/copy-wpf.ps1

```diff
@@ -30,13 +30,15 @@ function Print-Usage()
 Write-Host "*** Copy WPF files procedure ***"
 
 $RepoRoot = Resolve-Path (Join-Path $PSScriptRoot "..")
+Write-host "Repo root = " $RepoRoot
```

Comment:
It looks like some of these are debugging prints. Consider `Write-Debug` - https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-debug?view=powershell-7.4